### PR TITLE
Sidebar for dashboard mobile implementation with Ant Design

### PR DESCRIPTION
**Pull request by maidiance**
## Description

This PR focuses on sidebar minimization to allow the Dashboard to be more mobile friendly.
Co-authored by @SheebaHashmi 

#### Video Link

[Loom Video](https://www.loom.com/share/2755b21695e943da885c51c7a08546bb)

#### Trello Link

<blockquote class="trello-card"><a href="https://trello.com/c/Tu4JKBVc/297-dashboard-sidebar-minimized-in-mobile-view">Dashboard sidebar minimized in mobile view</a></blockquote>

## Type of change

- [✔️] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [✔️] My code follows the style guidelines of this project
- [✔️] I have performed a self-review of my own code
- [✔️] I have removed unnecessary comments/console logs from my code
- [✔️] My changes generate no new warnings
- [✔️] I have checked my code and corrected any misspellings
- [✔️] No duplicate code left within changed files
- [✔️] Size of pull request kept to a minimum
- [✔️] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by arsalan2243 — APPROVED** [review]
really good job fixing the side bar minimization issues. this can make it much more usable specially on small screens.
would you able to upload the video again? i couldn't find the Url.
overall, great job.

**Comment by maidiance** [comment]
> really good job fixing the side bar minimization issues. this can make it much more usable specially on small screens. would you able to upload the video again? i couldn't find the Url. overall, great job.
This was a draft when you first reviewed, but now the Loom is up :)

File: src/index.js [hunk 1]
```diff
@@ -19,10 +19,9 @@ import Signup from './components/pages/RoleSignup/Signup';
 import Mentee from './components/pages/RoleSignup/Applications/Mentee';
 import Mentor from './components/pages/RoleSignup/Applications/Mentor';
 
-import AppError from './components/pages/RoleSignup/Applications/AppError';
+import AppSuccess from './components/pages/RoleSignup/Applications/AppSuccess';
```
Comment (SheebaHashmi) [inline src/index.js:22]: Great job on fixing this error Winnie!

**Review by jonathan-mcgraw — APPROVED** [review]
Heck yeah, Winnie!! I know this has been giving FE a headache for awhile.

File: src/components/common/styles/Sidebar.css [hunk 2]
```diff
@@ -3,3 +3,18 @@
...
+.site-layout .site-layout-background {
+  height: 5%;
+  margin-top: 20%;
+  display: flex;
+  padding-top: 1%;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border-radius: 0 5px 5px 0;
+  width: 2% !important;
```
Comment (SheebaHashmi) [inline src/components/common/styles/Sidebar.css:16]: I like how you add custom styling to this component. Great work Winnie!

**Review by SheebaHashmi — APPROVED** [review]

File: src/components/common/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -218,6 +226,12 @@ const Sidebar = ({ children, userProfile }) => {
         </Menu>
       </Sider>
       <Layout className="site-layout">
+        <Sider className="site-layout-background">
+          {React.createElement(collapsed ? RightOutlined : LeftOutlined, {
```
Comment (paulstgermain) [inline src/components/common/Sidebar/Sidebar.js:230]: Smooth, smart implementation here!

**Pull request merged**